frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1084,6 +1104,13 @@ pub mod pallet {
 				Error::<T>::FallbackFailed
 			})?;
 
+			// transform BoundedVec<_, T::GovernanceFallback::MaxWinners> into
+			// `BoundedVec<_, T::MaxWinners>`
+			let supports: BoundedVec<_, T::MaxWinners> = supports
+				.into_inner()
+				.try_into()
+				.expect("integrity test guarantees both bounds are equal; qed");
```

Comment:
A defensive truncate could be a nice alternative, but this is also fine.

Reply:
more reasonable is actually to return an error, not panic. 

but defensively return an error e.g. `defensive_map_err`.